app/views/elements/courses/submenu.ctp

```diff
@@ -83,9 +71,29 @@ switch($submenu) {
         array('name' => 'Export Survey Group Sets', 'link' => "/surveygroups/export/$course_id")
     );
     break;
+  case "Canvas":
+    if ($canvasEnabled){
+        array_push(
+            $items,
+            array('name' => 'Import Students from Canvas',  'link' => "/users/import/$course_id/canvas")
+        );
+        array_push(
+            $items,
+            array('name' => 'Import Groups from Canvas', 'link' => "/groups/import/$course_id/canvas")
```

Comment:
With import /export groups as separate functions, the option `(Create New Groupset)` should be removed from dropdown when doing import.
![image](https://user-images.githubusercontent.com/13360325/38339766-da507f78-3824-11e8-9015-0ca6ec8bcfba.png)